ert_gui/tools/plot/plot_window.py

```diff
@@ -91,13 +120,25 @@ def currentPlotChanged(self):
             ):
                 self._updateCustomizer(plot_widget)
                 cases = self._case_selection_widget.getPlotCaseNames()
-                case_to_data_map = {
-                    case: self._api.data_for_key(case, key) for case in cases
-                }
+                case_to_data_map = {}
+                for case in cases:
+                    try:
+                        case_to_data_map[case] = self._api.data_for_key(case, key)
+                    except RequestError as e:
```

Comment:
Perhaps add a TimeoutError here aswell. I guess all api calls could result in a timeout